lib/drivers/native/driver_windows.go

```diff
@@ -0,0 +1,278 @@
+//go:build windows
+
+/**
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+package native
+
+// Native driver to run the workload on the host of the fish node
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/shirou/gopsutil/v3/cpu"
+	"github.com/shirou/gopsutil/v3/mem"
+
+	"github.com/adobe/aquarium-fish/lib/drivers"
+	"github.com/adobe/aquarium-fish/lib/log"
+	"github.com/adobe/aquarium-fish/lib/openapi/types"
+)
+
+// Implements drivers.ResourceDriverFactory interface
+type Factory struct{}
+
+func (f *Factory) Name() string {
+	return "native_win"
```

Comment:
Sorry, probably did not expressed that properly: I don't want those separated folders in native driver. We need to unify as much as possible - completely separated implementations is not what we need - that makes native driver too fragmented. I'm quite sure you already can put all those files together in one folder and try to find similarities between them and how to make them work together.

So you can use just "native" - and put this logic to some common file. And to separate the platforms we will use a bit different mechanism of "node_filter" to specify the platform in the label: https://github.com/adobe/aquarium-fish/blob/main/tests/node_filter_test.go